Dockerfile

```diff
@@ -12,23 +12,26 @@ RUN apt-get update && \
 
 # add group & user
 ARG USER_HOME=/home/piper
+ARG CFCLI_VERSION
+
 RUN addgroup -gid 1000 piper && \
     useradd piper --uid 1000 --gid 1000 --shell /bin/bash --home-dir "${USER_HOME}" --create-home && \
-    curl --location --silent "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=v7&source=github" | tar -zx -C /usr/local/bin && \
+    LATEST_CFCLI=$(curl -s https://api.github.com/repos/cloudfoundry/cli/releases/latest | grep tag_name | grep -v beta | cut -d \" -f 4 | tr -d v) && \
+    curl --location --silent "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=${CFCLI_VERSION:-$LATEST_CFCLI}&source=github" | tar -zx -C /usr/local/bin && \
     cf --version
 
 USER piper
 WORKDIR ${USER_HOME}
 
-ARG MTA_PLUGIN_VERSION=2.7.0
-ARG MTA_PLUGIN_URL=https://github.com/cloudfoundry-incubator/multiapps-cli-plugin/releases/download/v${MTA_PLUGIN_VERSION}/multiapps-plugin.linux64
-ARG CSPUSH_PLUGIN_VERSION=1.3.2
-ARG CSPUSH_PLUGIN_URL=https://github.com/dawu415/CF-CLI-Create-Service-Push-Plugin/releases/download/${CSPUSH_PLUGIN_VERSION}/CreateServicePushPlugin.linux64
+ARG MTA_PLUGIN_VERSION
+ARG CSPUSH_PLUGIN_VERSION
```

Comment:
you can still overwrite the versions in case you don't want to use the one from github via:
`docker build --build-arg MTA_PLUGIN_VERSION=2.7.0 --build-arg CSPUSH_PLUGIN_VERSION=1.3.2 ...`